e2e-tests/Jenkinsfile

```diff
@@ -34,7 +35,8 @@ pipeline {
         always {
           archiveArtifacts 'e2e-tests/results/*'
           junit 'e2e-tests/results/*.xml'
-          submitToActiveData('e2e-tests/results/py27.log')
+          submitToActiveData('e2e-tests/results/py27_raw.txt')
+          submitToTreeherder('socorro-tests', 'e2e', 'End-to-end integration tests', 'e2e-tests/results/*', 'e2e-tests/results/py27_tbpl.txt')
```

Comment:
Project should match the repository. In this case 'socorro' and not 'socorro-tests'.